client/src/Main.jsx

```diff
@@ -10,65 +10,55 @@ import React, { Component } from 'react';
 import Home from './Home';
 import Menu from './API/Menu';
 import Orders from './API/Order';
-import Login from './Auth/Login';
 import AppRoute from './index';
 import { BrowserRouter, Route, Link, Switch, Redirect } from 'react-router-dom';
 import { Button, Card, Row, Col, Navbar, NavItem } from 'react-materialize';
 import './css/general.css';
-import { Config, CognitoIdentityCredentials } from 'aws-sdk';
 import awsmobile from './aws-exports';
 import {Auth} from 'aws-amplify';
+import { Authenticator } from 'aws-amplify-react/dist/Auth';
```

Comment:
Authenticator was never used, it should be removed

Reply:
Done